### DISCUSSION TITLE
Feature request: custom keymapping

### DISCUSSION THREAD

**Discussion by UlyssesZh** [Genymobile/feedback#12] [Genymotion Desktop]
Really want custom mapping keyboard controls. See also: Genymobile/scrcpy#712

**Comment by agavignet** [comment]
Hi @UlyssesZh ,

Could you add some precision regarding your feature request? Does it concern Desktop Emulator window ? What kind of key mapping would you like? Can you provide us your use case?

Thank you for your request.
Best regards.

**Reply by UlyssesZh** [reply]
Actually Genymobile/scrcpy#712 has some comments that illustrated how the function may look like. What I want is just to be able to have profiles (so that we can change the mapping for different apps) of customized lists of mapping of keys to coordinates on the screen, and it should work well with multitouch at the same time.
My use case is use Genymobile to play games (either for fun or for testing what I am developing).
